### PR TITLE
[JUJU-560] Turns Jammy off as a supported series.

### PR DESCRIPTION
**Pull request by tlm**
Jammy has not been released yet and using Juju on any other platform
besides Linux will try and deploy Jammy.

## Checklist

 - [x] Requires a [pylibjuju](https://github.com/juju/python-libjuju) change
 - [x] Added [integration tests](https://github.com/juju/juju/tree/develop/tests) for the PR
 - [x] Added or updated [doc.go](https://discourse.jujucharms.com/t/readme-in-packages/451) related to packages changed
 - [x] Comments answer the question of why design decisions were made

## QA steps

N/A

## Documentation changes

N/A

## Bug reference

N/A

### CODE REVIEW

**Review by hpidcock — APPROVED** [review]

**Comment by tlm** [comment]
$$merge$$

**Comment by tlm** [comment]
$$merge$$

File: core/series/supportedseries.go [hunk 1]
```diff
@@ -4,6 +4,7 @@
 package series
 
 import (
+	"github.com/juju/juju/version"
```
Comment (hpidcock) [inline core/series/supportedseries.go:7]: fix grouping

File: core/series/supportedseries.go [hunk 2]
```diff
@@ -371,6 +372,28 @@ var (
 // the work to determine the latest lts series once.
 var latestLtsSeries string
 
+// DefaultSupportedLts return the default supported lts for this version of Juju if it is known, an LTS release and is
+// supported.
+func DefaultSupportedLts() (string, error) {
```
Comment (hpidcock) [inline core/series/supportedseries.go:377]: rename to DefaultSupportedLTS

File: core/series/supportedseries.go [hunk 3]
```diff
@@ -371,6 +372,28 @@ var (
...
+		return "", errors.NotSupportedf("default supported lts %q is not recognised as an LTS release", defaultLTS)
+	} else if !seriesVersion.Supported {
+		return "", errors.NotSupportedf("default supported lts %q", defaultLTS)
+	}
+
+	return defaultLTS, nil
+}
+
 // LatestLts returns the Latest LTS Release found in distro-info
 func LatestLts() string {
```
Comment (hpidcock) [inline core/series/supportedseries.go:398]: May as well fix this name too.

**Review by hpidcock — APPROVED** [review]

**Comment by tlm** [comment]
$$merge$$

**Comment by tlm** [comment]
$$merge$$

**Pull request merged**